client/src/components/TaskCard/TaskCard.js

```diff
@@ -35,6 +35,7 @@ const TaskCard = ({
 
   const reflectionDoneHandler = async (reflectionComments, satisfaction) => {
     await createTaskReflection(id, reflectionComments, satisfaction);
+    await toggleTaskHandler(id);
     toggleReflection();
```

Comment:
Ideally, we should do some type of error checking (in the event that the the task reflection couldn't be created or task couldn't be toggled. For our purposes, I think it's okay to have it as is.

Reply:
I should probably have noted this down but this component expects `toggleTaskHandler` to manage its own errors. If you look at `toggleTaskHandler` in `CalendarPage`, it does the error checking and sets the correct error messages to show.